src/nwnsdk/postgres/postgres_client.py

```diff
@@ -62,14 +59,46 @@ def store_job_result(self, job_id: uuid4, new_logs: str, new_status: JobStatus,
             stmnt = (
                 update(Job)
                 .where(Job.job_id == job_id)
-                .values(status=new_status.value, logs=new_logs, output_esdl=output_esdl, stopped_at=datetime.now())
+                .values(status=new_status, logs=new_logs, output_esdl=output_esdl, stopped_at=datetime.now())
             )
             session.execute(stmnt)
 
+    def get_job_status(self, job_id: uuid4) -> JobStatus:
+        LOGGER.debug("Retrieving job status for job %s", job_id)
+        with session_scope() as session:
+            stmnt = select(Job).options(load_only(Job.status)).where(Job.job_id == job_id)
+            job: Job = session.scalar(stmnt)
+            session.expunge(job)
+        return job.status
+
     def get_job(self, job_id: uuid4) -> Job:
-        session: Session
         LOGGER.debug("Retrieving job data for job %s", job_id)
         with session_scope() as session:
             stmnt = select(Job).where(Job.job_id == job_id)
             job: Job = session.scalar(stmnt)
+            session.expunge(job)
         return job
+
+    def get_jobs(self, job_ids: list[uuid4]) -> Job:
+        LOGGER.debug(f"Retrieving job data for jobs {','.join([str(job_id) for job_id in job_ids])}")
+        with session_scope() as session:
+            stmnt = (
+                select(Job)
+                .options(
+                    load_only(
+                        Job.job_id,
+                        Job.job_name,
+                        Job.work_flow_type,
+                        Job.user_name,
+                        Job.status,
+                        Job.added_at,
+                        Job.running_at,
+                        Job.stopped_at,
+                    )
+                )
+                .where(Job.job_id.in_(job_ids))
+            )
+
+            jobs = session.scalars(stmnt).all()
+            [session.expunge(job) for job in jobs]
```

Comment:
SQLAlchemy has an `expunge_all` https://docs.sqlalchemy.org/en/20/orm/session_api.html#sqlalchemy.orm.Session.expunge_all

Reply:
It is necessary else it would remove inserts/updates before committing.